app/controllers/shared/api.php

```diff
@@ -255,7 +251,64 @@
             }
         }
 
+        $route = $utopia->match($request);
+        $requestParams = array_combine(
+            array_keys($route->getParams()), array_column($route->getParams(), 'value')
+        );
```

Comment:
@shimonewman this will not work, we need the previous code, the route matched, doesn't have value field, that's only calculated during `execute` phase. so value here is always `null`

Reply:
The values is in `$utopia->$matches` which is not exposed, may be we should expose the matches? @eldadfux what od you think?

Reply:
With the latest framework changes the values should be populated. See this line: https://github.com/utopia-php/framework/blob/master/src/App.php#L644

Because we access the Route object in the shutdown every relevant variable should be accessible. Am I missing something?

Reply:
Never mind, I was using the old version of the framework. @eldadfux @shimonewman All good.

Reply:
Cool